tests/components/homewizard/snapshots/test_diagnostics.ambr

```diff
@@ -299,6 +299,180 @@
     }),
   })
 # ---
+# name: test_diagnostics[HWE-SKT-11]
```

Comment:
Considering the name change, I would have expected the old snapshots to be removed. 

You can force that by deleting the snapshots, and regenerate them from scratch